kubectl.zsh

```diff
@@ -2,12 +2,18 @@ setopt prompt_subst
 autoload -U add-zsh-hook
 
 function() {
-    local namespace separator modified_time_fmt
+    local namespace separator_left separator_right modified_time_fmt
 
     # Specify the separator between context and namespace
-    zstyle -s ':zsh-kubectl-prompt:' separator separator
-    if [[ -z "$separator" ]]; then
-        zstyle ':zsh-kubectl-prompt:' separator '/'
+    zstyle -s ':zsh-kubectl-prompt:' separator_left separator_left
+    if [[ -z "$separator_left" ]]; then
+        zstyle ':zsh-kubectl-prompt:' separator_left ':'
+    fi
```

Comment:
Removing this `separator` variable is a breaking change. Do not delete the `separator` variable. Here, if the `separator` variable is not empty, set the `separator_left` variable to the value of the `separator` variable.

```zsh
zstyle -s ':zsh-kubectl-prompt:' separator separator
if [[ -n "$separator" ]]; then
    zstyle ':zsh-kubectl-prompt:' separator_left "$separator"
fi

zstyle -s ':zsh-kubectl-prompt:' separator_left separator_left
if [[ -z "$separator_left" ]]; then
    zstyle ':zsh-kubectl-prompt:' separator_left '/'
fi
```

Also, please do not change the default value of the separator.